powheg.sh

```diff
@@ -1,11 +1,11 @@
 package: POWHEG
 version: "%(tag_basename)s"
-tag: "r3178-alice1"
+tag: "r3693-alice1"
 source: https://github.com/alisw/POWHEG
 requires:
   - fastjet
   - "GCC-Toolchain:(?!osx|slc5)"
-  - lhapdf5
+  - lhapdf-pdfsets
```

Comment:
@mfasDa Sorry, I only spotted this now and maybe I am overlooking something. Dropping lhapdf as a requirement here essentially results in it being only *build_*required since lhapdf-pdfsets only build_requires it. The latter is actually inconsistent with loading lhapdf from the lhapdf-pdfsets modulefile. If I remember correctly, not having lhapdf as a runtime requirement for lhapdf-pdfsets is done to avoid having to redeploy the pdfsets in case of an lhapdf update, correct? In this case, lhapdf should stay as a dependency here and we should fix the modulefile for lhapdf-pdfsets. If instead there is no good reason to have lhapdf only as build_requirement for lhapdf-pdfsets we should fix that. Could you. comment?

Reply:
@qgp Sorry, I overlooked this. As lhapdf-pdfsets loads LHAPDF in the modulefile I expected it was sufficient to set the dependency on lhapdf-pdfsets, but indeed lhapdf is only a build requirement for lhapdf-pdfsets, and the requirement here should be lhapdf (but no longer lhapdf5) + the lhapdf-pdfsets modulefile should be fixed. Maybe in this case also the herwig and SHERPA build recipes - which also use lhapdf6 - should be checked.

Reply:
@mfasDa How should we go ahead? Do you want to update the dependency here such that this PR can be merged?

Then, we fix the **lhapdf-pdfsets** modulefile and the dependencies for both **herwig** and **SHERPA** (I confirmed both list only lhapdf-pdfsets as requirement).

Reply:
HI @qgp, yes, let's do like this. I will fix the dependecies here.